src/decision/dMCDA.jl

```diff
@@ -360,12 +360,18 @@ function create_decision_matrix(
 
     # Wave damage, account for cases where no chance of damage or heat stress
     # if max > 0 then use damage probability from wave exposure
-    A[:, 4] .= maximum(wave_stress) != 0.0 ? (wave_stress .- minimum(wave_stress)) ./ (maximum(wave_stress) - minimum(wave_stress)) : 0.0
-    #A[:, 4] .= wave_stress ./ maximum(wave_stress)
+    A[:, 4] .= if maximum(wave_stress) != 0.0
```

Comment:
1. I personally think it's easier to read if-elses when they start with `A == B` than `A != B` (I know, it's a personal view. what do you think?)
2. Since you are using `maximum(wave_stress)` and `minimum(wave_stress)` a few timew, you could put them into variables and do this in one line (not a requirement, another personal preference, feel free to change only if you like it!)

```
    max_ws = maximum(wave_stress)
    min_ws = minimum(wave_stress)
    A[:, 4] .= 0.0 ? max_ws == 0.0 : (wave_stress .- min_ws) ./ (max_ws - min_ws)
```